BEIMA.Backend/AuthFunctions/Login.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using BEIMA.Backend.Models;
+using BEIMA.Backend.AuthService;
+using BEIMA.Backend.MongoService;
+using MongoDB.Bson.Serialization;
+using BCryptNet = BCrypt.Net.BCrypt;
+using MongoDB.Bson;
+using System.Collections.Generic;
+
+namespace BEIMA.Backend.AuthFunctions
+{
+    /// <summary>
+    /// Handles a request to login
+    /// </summary>
+    public static class Login
```

Comment:
I think we should add a unit test class for this as well.

Reply:
Brain short circuited, will get that added.